checker/manual/creating-a-checker.tex

```diff
@@ -14,7 +14,6 @@ \chapter{How to create a new checker\label{writing-a-checker}}
 If you only want to \emph{use} a checker, you do not need to read this
 chapter.
 
-
```

Comment:
Please try to avoid gratuitous whitespace changes.  They clutter the diff and then the history.